### DISCUSSION TITLE
리드미 구상 및 작성 논의

### DISCUSSION THREAD

**Discussion by jopopcorn** [euphony-io/check-in-speaker#27] [General]
@euphony-io/euphoria 
스피커 앱 소개와 사용 방법, 화면 캡처 등을 포함한 리드미를 간단하게 작성해보고 co-authors로 올리면 좋지 않을까 싶습니다!

화면 캡처는 개발이 마무리된 맨 마지막에 첨부해야 할 것 같아서 우선 가장 뒤로 미루고,
이외에 추가로 넣거나 수정할 부분 등이 있다면 자유롭게 코멘트 달아주세요!

**Comment by psh2849** [comment]
앱 소개랑 사용 방법, 간단한 기능 소개(텍스트), 간단한 사용 화면 캡처가 있으면 좋을 것 같고
처음 앱을 실행했을 때 입력해야하는 안심 번호를 어디서 확인할 수 있는지 간단하게나마 알려줄 수 있으면 좋을 것 같습니다!

**Reply by jopopcorn** [reply]
오오 안심번호 확인할 수 있는 방법 추가하는거 아주 좋습니다!

**Reply by jopopcorn** [reply]
더 추가할 사항을 고민해봤는데 Apache 2.0 라이센스도 추가하고 리드미에 명시하면 좋을 것 같네요!

**Comment by jopopcorn** [comment]
체크인 시작 버튼을 눌렀을 때 로딩 다이얼로그가 보여지는 화면을 제외하고는 모두 캡처했습니다!
로딩 다이얼로그도 적용되면 추가로 캡처 올리겠습니다~

<img width="336" alt="스크린샷 2021-10-04 오전 12 01 19" src="https://user-images.githubusercontent.com/35393459/135759786-5c120063-3d18-4343-90d6-4fd57b2781bc.png">

<img width="338" alt="스크린샷 2021-10-04 오전 12 01 37" src="https://user-images.githubusercontent.com/35393459/135759791-3c92ac49-3850-42ae-9c3b-94258d246703.png">

<img width="338" alt="스크린샷 2021-10-04 오전 12 04 45" src="https://user-images.githubusercontent.com/35393459/135759929-642f3627-ec46-4c69-9570-9c4668229976.png">

<img width="335" alt="스크린샷 2021-10-04 오전 12 46 28" src="https://user-images.githubusercontent.com/35393459/135761542-7788b258-8fdf-457b-8390-311962649ec7.png">
